src/Catalyst.Core.Modules.Web3/Controllers/MempoolController.cs

```diff
@@ -43,6 +43,13 @@ public MempoolController(IMempoolRepository<TransactionBroadcastDao> mempoolRepo
             _mempoolRepository = (MempoolRepository) mempoolRepository;
         }
 
+        [HttpGet("{id}")]
+        public TransactionBroadcastDao Get(string id)
+        {
+            id = id.ToLower();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4403188)